python_modules/dagster/dagster_tests/general_tests/check_tests/test_check.py

```diff
@@ -697,10 +697,10 @@ class Bar:
         check.inst(1, (Foo, Bar))
 
     with pytest.raises(CheckError, match=re.escape("not one of ['Bar', 'Foo']")):
-        check.inst(1, (Foo, Bar), desc="a desc")
+        check.inst(1, (Foo, Bar), additional_message="a desc")
 
     with pytest.raises(CheckError, match=re.escape("Desc: a desc")):
-        check.inst(1, (Foo, Bar), desc="a desc")
+        check.inst(1, (Foo, Bar), additional_message="a desc")
```

Comment:
[1] the pytest match of "Desc:" with `additional_message` set is what caught me.